pkg/controller/admissionchecks/provisioning/constants.go

```diff
@@ -19,8 +19,8 @@ package provisioning
 const (
 	ConfigKind             = "ProvisioningRequestConfig"
 	ControllerName         = "kueue.x-k8s.io/provisioning-request"
-	ConsumesAnnotationKey  = "cluster-autoscaler.kubernetes.io/consume-provisioning-request"
-	ClassNameAnnotationKey = "cluster-autoscaler.kubernetes.io/provisioning-class-name"
+	ConsumesAnnotationKey  = "autoscaling.x-k8s.io/consume-provisioning-request"
+	ClassNameAnnotationKey = "autoscaling.x-k8s.io/provisioning-class-name"
```

Comment:
I would not remove old annotations, since it won't be compatible with old CA versions. We need to support both annotations for some time. We can rename old annotations as `DeprecatedConsumesAnnotationKey` and `DeprecatedClassNameAnnotationKey`. Make sure that kueue adds both types of annotations.

Reply:
Sounds good, I'll fix it